### DISCUSSION TITLE
where the real-world tests cases can be downloads?

### DISCUSSION THREAD

**Discussion by zhemu26** [NOAA-EMC/WW3#1335] [Q&A]
I recently installed WAVEWATCH  3(6.07) and use it. However, on my Linux system ,i can not run ./model/bin/ww3_from/ftp.sh  so that i was failed to find the cases directory and there was no real-world test cases.

**Comment by zhemu26** [comment]
Where is the cases directory? I can not find this directory in my top directory. Do i need to download cases directory from elsewhere?

**Reply by Crystal5172117** [reply]
I encountered the same problem. May I ask how you solved it later?
